contact.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Contact Us</title>
+</head>
+<body>
+    <header>
+        <img src="images/" alt="dog logo">
+        <p>Seattle's priemer dog adoption agency</p>
+        <input type="button" value="Checkout">
+    </header>
+    <nav>
+        <ul>
+            <li><a href="index.html">Home /</a></li>
+            <li><a href="dogs.html">Dogs /</a></li>
+            <li><a href="blog.html">Blog</a></li>
+        </ul>
+    </nav>
+    <section>
+        <h1>Checkout</h1>
+        <form method="GET">
+            <label>
+                Name:
+                <input type="text" required name="name">
+            </label>
+            <label>
+                Email Address:
+                <input type="email" name="email">
+            </label>
+            <label>
+                Street Address:
+                <input type="text" name="street">
+            </label>
+            <label>
+                City:
```

Comment:
You may want to put the City, State, and Zip in a container to keep them on the same line.